Parser/pegen.c

```diff
@@ -938,8 +938,8 @@ _PyPegen_number_token(Parser *p)
         return NULL;
     }
 
-    return Constant(c, NULL, t->lineno, t->col_offset, t->end_lineno, t->end_col_offset,
-                    p->arena);
+    return Constant(c, NULL, (int) t->lineno, (int) t->col_offset, (int) t->end_lineno,
```

Comment:
Ugh..... I am not a fan of having to do these castings everywhere. I think we should either revert the Py_ssize_t change or adapt the AST C types to use Py_ssize_t instead of integers for these attributes. What do you think?

Reply:
Agreed. I'm not sure what I would go with. Probably just revert the `Py_ssize_t`s in the `Token` struct.

Reply:
Yeah, I think that would be better for now. I will investigate how much would entail adapting the AST structs, but that likely will need more discussion.

Reply:
Right! I'll close this PR then and open a new one.